src/sql/src/session/vars/definitions.rs

```diff
@@ -1785,6 +1785,12 @@ macro_rules! feature_flags {
 }
 
 feature_flags!(
+    {
+        name: enable_let_prefix_extraction,
+        desc: "Enables hoisting of loop-invariant CTE bindindgs",
+        default: true,
+        enable_for_item_parsing: false,
+    },
```

Comment:
(We usually add them at the end of the list, but no need to go through another CI cycle just for this.)

Reply:
I plan to delete it immediately. Tbh, might be best to group them that way?